packages/gallery/src/components/gallery/proGallery/galleryContainer.js

```diff
@@ -130,6 +131,7 @@ export class GalleryContainer extends React.Component {
   }
 
   componentDidMount() {
+    windowWrapper.isGalleryMounted = true;
```

Comment:
wrap this in a function and think of the context. isGalleryMounted is the gallery's bussiness, not a windowWrapper logic. so the windowWrapper might have a flag that says useMock(or something like this) that could change to false after the gallery calls a function.